Ugizwenayo-Divine/FoodStuffPriceCompare-App
# chore(setupHoundCi): setting up hound CI

#### What does this PR do?
- setting up hound CI
#### Description of Task to be completed?
- added the git repo to hound
- created .hound.yml file
- configured the .hound.yml file
- got the reviewed by hound badge and putted it in readme file
#### How should this be manually tested?
- After cloning the project repo, run npm i inside the cloned repo folder
- change in cloned repo
- git fetch
- git checkout ch-setup-babel
- npm i
- you must the reviewed by Hound badge
- see the .hound.yml file
#### Any background context you want to provide?
- you must have node installed locally
#### What are the relevant pivotal tracker stories?
N/A
#### Screenshots (if appropriate)
N/A
